src/monattack.cpp

```diff
@@ -122,6 +125,9 @@ void mattack::shriek(monster *z, int index)
 
 void mattack::howl(monster *z, int index)
 {
+    if( z->friendly ) {
+        return; // TODO: handle friendly monsters
```

Comment:
Chance of friendlying other howlers in the area, I'd imagine: wolves use howls for communication and can convey that the ape is on Team Wolf.